azure-automation-plus-dsc-lab/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUserName": {
+      "value": "adm.infra.user"
+    },
+    "adminPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "studentRandomInfix": {
+      "value": "GEN-UNIQUE-8"
+    },
+    "studentNumber": {
+      "value": 10
```

Comment:
remove this and use a defaultValue in the template

Reply:
Acknowledged. Done.